bin/taste-tester

```diff
@@ -434,6 +435,10 @@ MODES:
     TasteTester::Hooks.get(path)
   end
 
+  if ARGV && [:test, :untest].include?(mode.to_sym)
+    TasteTester::Config.servers.concat ARGV
```

Comment:
So now some of your servers are in `TasteTester::Config` and some are in `options[:server]`? Doesn't seem useful. Am I missing the rest of the code where you actually implement looking at `TasteTester::Config.servers`?

In general I dislike this approach. It's brittle.

If we wanted to do something like this, I'd instead suggest accepting `-` as an arg to `-s`, and if that's there reading hostnames from STDIN.

Reply:
Those are merged 10 lines above (424 or 425). If you prefer I can move this before the merge and `concat` to `options[:servers]`. Don't really see how that makes any difference, but I guess that means I can remove the default in `TasteTester::Config`
And I know you dislike that approach, you've been refusing to do this for years even though many users have asked for it. I've been using this code for a while and others mentioned that it could be useful for them.

Reply:
There is no changes on lines 424 or 425. Are you missing changes in your PR? The only other changes in this file in this PR are in the help.

Reply:
Oh. I see.

I'll let others weigh in. I'm open to the idea of easier ways of passing more servers in, but the idea you can mix-n-match like this is just confusing to users. It's non-standard... I can't think of another utility that works like this. Where as `-s -` is a pretty common pattern.

I'd like it to continue to look like a unix utility, not a babar-specific utility.